packages/xfsprogs/xfsprogs.spec

```diff
@@ -0,0 +1,93 @@
+Name: %{_cross_os}xfsprogs
+Version: 6.3.0
+Release: 1%{?dist}
+Summary: Utilities for managing the XFS filesystem
+License: GPL+ and LGPLv2+
```

Comment:
This package includes copies of the GPL-2.0-only and LGPL-2.1-only licenses, this should match those licenses.

Reply:
I'll take a look and make sure this is right.